src/f32ext/utils.rs

```diff
@@ -1,8 +1,11 @@
 use core::f32;
+use core::i32;
 pub const SIGN_MASK: u32 = 0b10000000_00000000_00000000_00000000;
 pub const EXPONENT_MASK: u32 = 0b01111111_10000000_00000000_00000000;
 pub const MANTISSA_MASK: u32 = 0b00000000_01111111_11111111_11111111;
 pub const EXPONENT_BIAS: u32 = 127;
+//core::f32 has mantissa digits, but the actual occupied bits are 24 - 1.
+pub const MANTISSA_BITS: u32 = 23;
```

Comment:
Looks like this is unused?

Reply:
Oops forgot to run clippy, it's been a while since I've beenable to work with rust